### PR TITLE
Fix/send responses everywhere customInputs were being sent.

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Adds responses everywhere customInputs was being sent earlier

**Environment**:  Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- [x] Test different scenarios like Request Reschedule(Email, Calendar, WEbhook) Reschedule(Email, Calendar, WEbhook), Cancellation(Email, Calendar, WEbhook), Reminder Emails

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: packages/lib/getBooking.tsx [hunk 1]
```diff
@@ -14,7 +14,6 @@ type BookingSelect = {
     };
   };
   location: true;
-  smsReminderNumber: true;
```
Comment (hariombalhara) [inline packages/lib/getBooking.tsx:17]: wasn't needed

File: packages/trpc/server/routers/viewer/bookings.tsx [hunk 2]
```diff
@@ -774,21 +776,6 @@ export const bookingsRouter = router({
       },
     });
 
-    const bookingFields = bookingRaw.eventType
-      ? getBookingFieldsWithSystemFields(bookingRaw.eventType)
-      : null;
-
-    const booking = {
-      ...bookingRaw,
-      responses: bookingResponsesDbSchema.parse(bookingRaw.responses),
```
Comment (hariombalhara) [inline packages/trpc/server/routers/viewer/bookings.tsx:783]: No need to parse here. getCalEventResponses can handle the parsing itself now.

File: packages/features/bookings/lib/getCalEventResponses.ts [hunk 3]
```diff
@@ -55,5 +77,5 @@ export const getCalEventResponses = ({
       };
     }
   }
-  return { calEventUserFieldsResponses, calEventResponses };
+  return { userFieldsResponses: calEventUserFieldsResponses, responses: calEventResponses };
```
Comment (hariombalhara) [inline packages/features/bookings/lib/getCalEventResponses.ts:80]: It allows us to simply spread over the return value to build cal event.

**Review by CarinaWolli — COMMENTED** [review]
It's also missing for the endpoint to test our zapier trigger. It's this endpoint: https://github.com/calcom/cal.com/blob/main/packages/app-store/zapier/api/subscriptions/listBookings.ts

Could you add that too?

**Comment by hariombalhara** [comment]
@CarinaWolli Thanks !! I have added it at that place also now.

File: pr-progress/pr-progress.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# PR Progress Updates
```
Comment (CarinaWolli) [inline pr-progress/pr-progress.md:1]: we should remove that file before merging, @hariombalhara can you make sure that the found bugs are not getting lost and issues are created for them?

File: packages/features/webhooks/lib/sendPayload.ts [hunk 5]
```diff
@@ -46,6 +46,7 @@ function getZapierPayload(data: CalendarEvent & EventTypeInfo & { status?: strin
     description: data.description,
     customInputs: data.customInputs,
     responses: data.responses,
+    userFieldsResponses: data.userFieldsResponses,
```
Comment (CarinaWolli) [inline packages/features/webhooks/lib/sendPayload.ts:49]: I pushed that change. We need to have the same payload in the zapier payload and in the /listBookings endpoint

**Review by CarinaWolli — COMMENTED** [review]
LGTM! I pushed a change to add userFieldsResponses to the Zapier payload. Will approve after pr-progress.md is removed

File: pr-progress/pr-progress.md (hunk 4 above)
Comment (hariombalhara) [reply pr-progress/pr-progress.md]: Done and filed bugs. One of the fixes got merged today itself.

**Review by CarinaWolli — APPROVED** [review]
Nice, looking good now 🙌🏻

**Pull request merged**